### ISSUE TITLE
How can I implement a dlsvm layer with Keras?

### ISSUE THREAD

**Issue by laubonghaudoi** [fchollet/keras#4152]
I have googled for code samples of dlsvms introduced in this paper http://www.cs.toronto.edu/~tang/papers/dlsvm.pdf.
yet I got no results. So how can I replace the last softmax layer with a svm classifier?

**Comment by stenbln** [comment]
You could replace it but note that for these small improvements (as paper suggests) the whole network needs to be retrained instead of just attaching dlsvm layer at the end.
